.github/workflows/danger.yml

```diff
@@ -0,0 +1,18 @@
+name: Danger CI
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Danger
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          npm install --save-dev @babel/plugin-transform-flow-strip-types
```

Comment:
is `@babel/plugin-transform-flow-strip-types` mandatory to run danger? I'm wondering if there's a local package.json/babel config we should apply this to as well :thinking:

Reply:
yes, without this line, I got an error, see https://github.com/vector-im/element-android/runs/7524822865 👍 

```shell
Run danger/danger-js@9.1.6
/usr/bin/docker run --name cd98fa901ef5630854d6b946f9faf39f02a4d_20631b --label 4cd98f --workdir /github/workspace --rm -e GITHUB_TOKEN -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_RUN_ATTEMPT -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_REF_NAME -e GITHUB_REF_PROTECTED -e GITHUB_REF_TYPE -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e GITHUB_STEP_SUMMARY -e RUNNER_OS -e RUNNER_ARCH -e RUNNER_NAME -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/element-android/element-android":"/github/workspace" 4cd98f:a901ef5630854d6b946f9faf39f02a4d
Error:  { Error: Cannot find module '@babel/plugin-transform-flow-strip-types' from '/github/workspace'
    at Function.module.exports [as sync] (/usr/src/danger/node_modules/resolve/lib/sync.js:40:15)
    at resolveStandardizedName (/usr/src/danger/node_modules/@babel/core/lib/config/files/plugins.js:101:31)
    at resolvePlugin (/usr/src/danger/node_modules/@babel/core/lib/config/files/plugins.js:54:10)
    at loadPlugin (/usr/src/danger/node_modules/@babel/core/lib/config/files/plugins.js:62:20)
    at createDescriptor (/usr/src/danger/node_modules/@babel/core/lib/config/config-descriptors.js:154:9)
    at items.map (/usr/src/danger/node_modules/@babel/core/lib/config/config-descriptors.js:109:50)
    at Array.map (<anonymous>)
    at createDescriptors (/usr/src/danger/node_modules/@babel/core/lib/config/config-descriptors.js:109:29)
    at createPluginDescriptors (/usr/src/danger/node_modules/@babel/core/lib/config/config-descriptors.js:105:10)
    at alias (/usr/src/danger/node_modules/@babel/core/lib/config/config-descriptors.js:63:49) code: 'MODULE_NOT_FOUND' }
```

Reply:
Actually this is more https://github.com/vector-im/element-android/runs/7526127685 :

```shell
Run danger/danger-js@11.1.1
/usr/bin/docker run --name cd98fbc3b34cafe1d46bbabf9cc41d533b8f8_c74665 --label 4cd98f --workdir /github/workspace --rm -e GITHUB_TOKEN -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_RUN_ATTEMPT -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_REF_NAME -e GITHUB_REF_PROTECTED -e GITHUB_REF_TYPE -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e GITHUB_STEP_SUMMARY -e RUNNER_OS -e RUNNER_ARCH -e RUNNER_NAME -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/element-android/element-android":"/github/workspace" 4cd98f:bc3b34cafe1d46bbabf9cc41d533b8f8
Error:  Error: Cannot find module '@babel/plugin-transform-flow-strip-types'
Require stack:
- /usr/src/danger/node_modules/@babel/core/lib/config/files/plugins.js
- /usr/src/danger/node_modules/@babel/core/lib/config/files/index.js
- /usr/src/danger/node_modules/@babel/core/lib/index.js
- /usr/src/danger/distribution/runner/runners/utils/transpiler.js
- /usr/src/danger/distribution/runner/runners/inline.js
- /usr/src/danger/distribution/commands/danger-runner.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at resolve (internal/modules/cjs/helpers.js:107:19)
    at resolveStandardizedName (/usr/src/danger/node_modules/@babel/core/lib/config/files/plugins.js:111:7)
    at resolvePlugin (/usr/src/danger/node_modules/@babel/core/lib/config/files/plugins.js:55:10)
    at loadPlugin (/usr/src/danger/node_modules/@babel/core/lib/config/files/plugins.js:63:20)
    at loadPlugin.next (<anonymous>)
    at createDescriptor (/usr/src/danger/node_modules/@babel/core/lib/config/config-descriptors.js:187:16)
    at createDescriptor.next (<anonymous>)
    at evaluateSync (/usr/src/danger/node_modules/gensync/index.js:251:28)
    at /usr/src/danger/node_modules/gensync/index.js:31:34 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/src/danger/node_modules/@babel/core/lib/config/files/plugins.js',
    '/usr/src/danger/node_modules/@babel/core/lib/config/files/index.js',
    '/usr/src/danger/node_modules/@babel/core/lib/index.js',
    '/usr/src/danger/distribution/runner/runners/utils/transpiler.js',
    '/usr/src/danger/distribution/runner/runners/inline.js',
    '/usr/src/danger/distribution/commands/danger-runner.js'
  ]
}
```